homeassistant/components/tailwind/binary_sensor.py

```diff
@@ -0,0 +1,78 @@
+"""Binary sensor entity platform for Tailwind."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from gotailwind.models import TailwindDoor
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import TailwindDataUpdateCoordinator
+from .entity import TailwindDoorEntity
+
+
+@dataclass(kw_only=True, frozen=True)
+class TailwindDoorBinarySensorEntityDescription(BinarySensorEntityDescription):
+    """Class describing Tailwind door binary sensor entities."""
+
+    is_on_fn: Callable[[TailwindDoor], bool]
+
+
+DESCRIPTIONS: tuple[TailwindDoorBinarySensorEntityDescription, ...] = (
+    TailwindDoorBinarySensorEntityDescription(
+        key="locked_out",
+        translation_key="operational_status",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:garage-alert",
+        is_on_fn=lambda door: not door.locked_out,
```

Comment:
What does locked out mean in this context? Normally we want the baseline secure state to be off for doors and windows and similar things.

Reply:
Locked out, means that the door is locked out for remote operation.

This is part of the UL certifications/requirements/standards. Tailwind is one of the few that is certified.

Something happened/was detected that locks out remote operation. Manual operation on the physical door is required (open & close it once) to ensure everything is clear.

Reply:
Then I suggest we invert the condition so that the sensor is on when it requires attention from the user as that's how we normally define binary sensors.

Reply:
I doubted, hence me flipping it in a later commit. Will revert / flip it around in a follow up PR.